premake5.lua

```diff
@@ -262,7 +262,7 @@ workspace "slang"
         optimize "Off"
         symbols "On"
         defines { "_DEBUG" }
-
+        -- staticruntime "Off"
```

Comment:
I'm assuming this is so that we use dll CRT when debug so we get the superior leak tracking.